frappe/printing/page/print_format_builder/print_format_builder.css

```diff
@@ -73,6 +73,7 @@
 	top: 110px;
 	bottom: 0px;
 	overflow-y: auto;
+	left: 0px;
```

Comment:
Breaks the UI in non-RTL mode.

![image](https://cloud.githubusercontent.com/assets/9355208/26677819/13675754-46eb-11e7-91c4-0b9e2b06c721.png)

Add it to `desk-rtl.css`